java-checks-testkit/src/main/java/org/sonar/java/checks/verifier/CheckVerifier.java

```diff
@@ -202,7 +202,7 @@ private static boolean isLinear(AnalyzerMessage issue) {
       String json = Resources.toString(resource, Charsets.UTF_8);
       return LINEAR_FUNC_PATTERN.matcher(json).find();
     } catch (IOException e) {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1166)